app/century-scale-storage/index.html

```diff
@@ -2365,6 +2353,7 @@ <h2>
                         </g>
                     </svg>
                 </a>
+                <p> At the time of publication, this piece referred to a 2021 request for video footage of a lecture by Admiral Grace Hopper present in the National Security Agency’s archives, which was <a href="https://www.muckrock.com/news/archives/2024/jul/10/grace-hopper-lost-lecture-found-nsa/">denied</a> as the agency no longer owned a machine capable of playing back AMPEX video tapes. In August 2024, the NSA <a href="https://www.nsa.gov/Press-Room/Press-Releases-Statements/Press-Release-View/Article/3884041/nsa-releases-copy-of-internal-lecture-delivered-by-computing-giant-rear-adm-gra/">released</a> a digitized version of the tapes, with assistance from the National Archives and Records Administration. The piece has been updated to reflect the release. A preserved copy of the earlier version can be seen <a href="https://perma.cc/6YV9-T87P">via Perma.cc</a>.</p>
```

Comment:
This will set the added paragraph in italics:

```suggestion
                <p><em>At the time of publication, this piece referred to a 2021 request for video footage of a lecture by Admiral Grace Hopper present in the National Security Agency’s archives, which was <a href="https://www.muckrock.com/news/archives/2024/jul/10/grace-hopper-lost-lecture-found-nsa/">denied</a> as the agency no longer owned a machine capable of playing back AMPEX video tapes. In August 2024, the NSA <a href="https://www.nsa.gov/Press-Room/Press-Releases-Statements/Press-Release-View/Article/3884041/nsa-releases-copy-of-internal-lecture-delivered-by-computing-giant-rear-adm-gra/">released</a> a digitized version of the tapes, with assistance from the National Archives and Records Administration. The piece has been updated to reflect the release. A preserved copy of the earlier version can be seen <a href="https://perma.cc/6YV9-T87P">via Perma.cc</a>.</em></p>
```